src/Microsoft.SqlTools.ServiceLayer/ObjectManagement/ObjectTypes/DatabaseRole/DatabaseRoleViewInfo.cs

```diff
@@ -0,0 +1,15 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+namespace Microsoft.SqlTools.ServiceLayer.ObjectManagement
+{
+    /// <summary>
+    /// a class for storing various database role view properties
+    /// </summary>
+    public class DatabaseRoleViewInfo : SqlObjectViewInfo
+    {
+        public string[]? Schemas { get; set; }
```

Comment:
we should also have an IsFixedRole property, right?

Reply:
also need following properties for the owner picking experience to work.
1. Users
2. Database Roles
3. Application Roles

Reply:
actually, I'll use the object search API to get these objects.

Reply:
IsFixedRole - Yes, I missed this one, will add.
Yes I was also planning to do that via the obj search API